modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= S3 - Compatible backup storage providers
+
```

Comment:
Should be "S3 compatible" and probably "AWS S3 compatible" throughout.  Search for "S3 com" in https://redhat-documentation.github.io/supplementary-style-guide/#project